### ISSUE TITLE
Update SSL certificate on the fly

### ISSUE THREAD

**Issue by naanlizard** [AirenSoft/OvenMediaEngine#359]
**Is your feature request related to a problem? Please describe.**
Downtime when updating an SSL certificate is bad for users. 

**Describe the solution you'd like**
Some mechanism to update the SSL certificate while the server is live, without interrupting streams, would be ideal. Something like `nginx -s reload` basically

**Describe alternatives you've considered**
The only real alternative is scheduled maintenance once every 6 months for hard downtime, which we prefer to avoid.

**Comment by basisbit** [comment]
You'll have to restart your server for security updates more often than once every few months anyways, thus I suggest you to set up more than one edge server and use your loadbalancer to send the users to the desired server/servers.

**Comment by dimiden** [comment]
@naanlizard I agree that restarting daemon for TLS certificate replacement is not a good idea.
OME uses ```SIGHUP``` signal for config update (now only supports reloading ```logger.xml```), and I will consider improving TLS certificate to reload when this signal occurs.

**Comment by naanlizard** [comment]
Is there any action on this? Any way I can help?

**Comment by basisbit** [comment]
+1

**Comment by naanlizard** [comment]
@getroot where in the priorities list is this? It's the last feature we're waiting on for feature parity with our current setup, and hopefully not terribly difficult. Even better if it automatically updates the cert if the file changes on the disk :)

**Comment by getroot** [comment]
This feature is very low priority as it is needed about once a year and often restarts OME to update OvenMediaEngine. I will soon organize and share my priority tasks for 2023.
I wish you a happy year!

**Comment by naanlizard** [comment]
I guess we'll have to update our maintenance schedules for some planned downtime at least every few months, not the worst I suppose but not great
